### PR TITLE
GH-1 : Frontend for primitive posts

### PR DESCRIPTION
**Pull request by lucasdrufva**
A primitive fronend for displaying list of posts and allowing user to submit a post containing only a title

### CODE REVIEW

File: front-end/script.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+const BASE_SERVER = "http://localhost:8080"
+
+document.getElementById("submit").onclick = () => {
```
Comment (Semillita) [inline front-end/script.js:3]: Change to event listener

**Review by Semillita — CHANGES_REQUESTED** [review]
Mostly good

File: front-end/script.js (hunk 1 above)
Comment (lucasdrufva) [reply front-end/script.js]: fixed?

**Review by Semillita — APPROVED** [review]
Good

**Pull request merged**